alsongDalsong/alsongDalsong/alsongDalsong/Sources/Views/SelectMusic/SelectMusicViewModel.swift

```diff
@@ -74,6 +77,23 @@ final class SelectMusicViewModel: ObservableObject, @unchecked Sendable {
             .store(in: &cancellables)
     }
     
+    private func bindSearchTerm() {
+        $searchTerm
+            .debounce(for: .milliseconds(500), scheduler: DispatchQueue.main)
+            .sink { [weak self] term in
+                guard let self else { return }
+                if term.isEmpty { resetSearchList() }
+                else {
+                    Task {
+                        do {
+                            try await self.searchMusic(text: term)
+                        } catch {}
+                    }
+                }
```

Comment:
어자피 줄바꿈이 있으니 이렇게 쓰시는거 어떤가요??

```suggestion
                if term.isEmpty { 
                    resetSearchList() 
                } else {
                    Task {
                        do {
                            try await self.searchMusic(text: term)
                        } catch {}
                    }
                }
```

Reply:
if term.isEmpty { resetSearchList() ; return } 
으로 먼저 early return 하는것도 좋아보입니당